server/services/team/service.ts

```diff
@@ -518,12 +518,12 @@ export class TeamService extends ServiceClass implements ITeamService {
 			};
 		}
 
-		const cursor = this.TeamMembersModel.findMembersInfoByTeamId(teamId, count, offset, query);
+		const cursor = this.TeamMembersModel.findMembersInfoByTeamId(teamId, count, offset);
 
 		const records = await cursor.toArray();
 		const results: ITeamMemberInfo[] = [];
 		for await (const record of records) {
-			const user = await this.Users.findOneById(record.userId);
+			const user = await this.Users.findOne({ ...query, _id: record.userId });
```

Comment:
Why was the change needed here?

Reply:
I've just changed the way the `query` is used here. It may be working better now 😄